sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StreamingWriteFn.java

```diff
@@ -70,6 +73,15 @@
   /** Tracks bytes written, exposed as "ByteCount" Counter. */
   private Counter byteCounter = SinkMetrics.bytesWritten();
 
+  private Set<KV<String, String>> metricFilter =
+      ImmutableSet.of(
+          KV.of(
+              LatencyRecordingHttpRequestInitializer.HISTOGRAM_URN.split(":", 2)[0],
```

Comment:
It looks like you are parsing the URN and treating it as if it were the "namespace" and "name" of the metric.

However, this metric does not have a namespace+name, and should not be modeled as such.

Remember, namespace+name is a concept that only applies to user defined metrics. The namespace and name are simply labels populated with the keys "namespace" and "name"
See the MonitoringInfoSpec on USER_SUM_INT64
https://github.com/apache/beam/blob/eca935bc36036dd11c5f2bd20052b08049375bfc/model/pipeline/src/main/proto/metrics.proto#L69

This metric should be identified using the urn, and it's labels.
Instantiate a MonitoringInfoMetricName for it instead, and update the comparison logic to just filter using MetricNames (MonitoringInfoMetricName is a subclass of MetricName).

Reply:
Done.